rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -84,22 +90,24 @@ s32 npDrmIsAvailable(vm::cptr<u8> k_licensee_addr, vm::cptr<char> drm_path)
 	auto edatkeys = fxm::get_always<EdatKeys_t>();
 
 	// Search for a compatible RAP file. 
-	for (const auto& entry : fs::dir(rap_lpath))
-	{
-		if (entry.name.find(title_id) != -1)
+	if (!k_licensee_addr) {
```

Comment:
Honestly im not sure why you are flagging these all up when its a base commit for another pr..